cypress/e2e/user-flows/scenario_slice_association.cy.js

```diff
@@ -0,0 +1,121 @@
+const random = Date.now();
+
+const customTypeName = `My Test ${random}`;
+const customTypeId = `my_test_${random}`;
+
+const sliceName = `TestSlice${random}`;
+const sliceId = `test_slice${random}`; // generated automatically from the slice name
+const sliceLib = "slices";
+
+describe("I am an existing SM user (Next) and I want to associate a Slice to a CT and review my experience.", () => {
```

Comment:
https://jestjs.io/docs/setup-teardown#order-of-execution

I believe inside the describe itself, the `it` will run one by one waiting for the the previous one to finish.

Reply:
One scenario could go in one `it`, but then it might be tougher to trace back issues as we have bigger steps in the scenario itself.

I'd be ok to change it but I guess we should do it for all the existing scenarios.